### PR TITLE
Use rhtap-service-push release pipeline

### PR DESCRIPTION
**Pull request by scoheb**
- Use for rhtap-build and rhtap-gitops
- Add new external-secrets needed for introduced tasks

### CODE REVIEW

File: ci/teams/rhtap-servicerelease/release-external-secrets.yaml [hunk 1]
```diff
@@ -9,4 +9,4 @@ spec:
   application: external-secrets
   displayName: ''
   origin: rhtap-gitops-tenant
-  releaseStrategy: push-to-external-registry-strategy
+  releaseStrategy: rhtap-service-push-strategy
```
Comment (sbose78) [inline ci/teams/rhtap-servicerelease/release-external-secrets.yaml:12]: Is the new strategy a superset of the old one?

Comment (scoheb) [reply ci/teams/rhtap-servicerelease/release-external-secrets.yaml:12]: yes! adds the PR creation and optional slack notification

**Review by sbose78 — APPROVED** [review]

File: ci/teams/rhtap-servicerelease/release-build.yaml [hunk 2]
```diff
@@ -10,4 +10,12 @@ spec:
   application: build
   displayName: ''
   origin: rhtap-build-tenant
-  releaseStrategy: push-to-external-registry-strategy
+  releaseStrategy: rhtap-service-push-strategy
+  extraData:
```
Comment (sbose78) [inline ci/teams/rhtap-servicerelease/release-build.yaml:14]: and we have slack notification enabled for build only?

Comment (scoheb) [reply ci/teams/rhtap-servicerelease/release-build.yaml:14]: to start yes. gitops does not have a webhook defined in slack secret.

**Pull request merged**